### DISCUSSION TITLE
5 broken links in CHANGELOG.md ?

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#604] [General]
The 5 links in CHANGELOG.md that start with https://github.com/valinet/ExplorerPatcher#
link only to the homepage https://github.com/valinet/ExplorerPatcher.

They are all in the section ## 22000.194.0.20.

For example:
CHANGELOG.md
Tested on build: 22000.194.
 400 * Removed the option to disable the modern search box in File Explorer. Instead, you now run a command which disables it globally on your user account (works in "Open" dialogs as well); read [here](https://github.com/valinet/ExplorerPatcher#disable-the-modern-search-box-in-File-Explorer)

**Comment by valinet** [comment]
The CHANGELOG is for reference purposes. Some things in there have changed and are even obsolete now. It mainly serves as a historic resource. v22000.194 was quite a while ago. No need to be so pedantic.

**Reply by chip33** [reply]
you could just delete the links then if you like, It confused me as I was trying to find the links to read up on what it was referencing.

**Reply by valinet** [reply]
Not really, the change log is historic, I treat it as "append only".

**Reply by chip33** [reply]
okay :)
